biometrics/extract.py

```diff
@@ -211,8 +209,11 @@ def _pileup(self, bam, site):
                 # happens sometimes from fgbio duplex sequening toolset
                 try:
                     base_qual = pileupread.alignment.qual[pileupread.query_position]
+                    #this is a character value
                 except:
-                    base_qual = 30
+                    base_qual = chr(0+33)
```

Comment:
Sorry I am not sure I understand why we would do `0+33`, I believe if the base_qual is obtained using `pileupread.alignment.qual[pileupread.query_position]`  is an integer value and in certain cases, we dont get the integer value and it is assigned some kind of string. I believe we either set in those scenario the value as 30 or we calculate avg. quality of the read itself as a default value in those cases.